### DISCUSSION TITLE
I have a huge bill from AWS we are just testing...

### DISCUSSION THREAD

**Discussion by RoryClearSky** [ant-media/Ant-Media-Server#6835] [FAQs]
We are testing Ant Media, I subscribed to their 99.00 monthly recurring fee.  Tested about 2 hours and Ant Media is charging me 350 US.  What am i doing wrong.

**Comment by muratugureminoglu** [comment]
Hi @RoryClearSky 

Can you please contact support[@]antmedia.io so I can check this?

Also, if you share the following details, we can reach a faster conclusion.

1. The email address from which you made the purchase
2. Where do you use AMS ? Local, Cloud (AWS, Azure, GCP etc.)
3. If you are using it in the cloud, check if you are using a Marketplace product. 

Thanks.
